docs/installation/docker_image.rst

```diff
@@ -10,7 +10,7 @@ In order to be able to download the Gemnasium Enterprise docker image from the `
 authenticated on docker hub, and provide the username used to pull the image to
 Gemnasium.
 
-.. todo: We'll probably leave the gemnasium/enterprise image public, to simplify the installation
+To have access to the Gemnasium Enterprise image, you need to have a license from us, and we will then give you access to it via the Docker Hub. It is a private image.
```

Comment:
Not really accurate: actually, we will authorize clients to access to our docker image, but they need to provide a  docker user for that. The licence sent will permit to run and sync with our servers.